perfkitbenchmarker/aws/aws_virtual_machine.py

```diff
@@ -266,22 +266,22 @@ def CreateScratchDisk(self, disk_spec):
     Args:
       disk_spec: virtual_machine.BaseDiskSpec object of the disk.
     """
-    volume = aws_disk.AwsDisk(disk_spec, self.zone)
-    self.scratch_disks.append(volume)
-
-    if volume.disk_type == disk.LOCAL:
-      if self.local_drive_counter >= self.max_local_drives:
+    # Instantiate the disk(s) that we want to create.
+    if disk_spec.disk_type == disk.LOCAL:
```

Comment:
Strip disks doesn't have to be local ssds. aws set max iops for single gp2 disk, it is useful if we can strip multiple gp2 disks together.